README.md

```diff
@@ -2,18 +2,25 @@
   <img width="60px" src="https://user-images.githubusercontent.com/6180201/124313197-cc93f200-db70-11eb-864a-fc65765fc038.png" alt="giant microphone"/>   
  <br/>
   <h2 align="center">Whisper Playground</h2>
-  <h6 align="center">Instantly build speech2text apps in 99 languages using OpenAI's Whisper</h6>
-  <h6 align="center"> Live demo https://whisperui.monsterapi.ai</h6>
+  <h6 align="center">Instantly build real-time speech2text apps in 99 languages using OpenAI's Whisper, Diart, and Pyannote</h6>
+  <h6 align="center">Live demo out soon!</h6>
 </div>
 
 ![visitors](https://visitor-badge.glitch.me/badge?page_id=saharmor.whisper-playground&left_color=green&right_color=red)
 
-https://user-images.githubusercontent.com/6180201/194214280-3a4106d5-53cf-4463-ba0e-201d8e1b0c0e.mp4
+Sequential Demo:
+
+https://github.com/ethanzrd/whisper-playground/assets/79014814/34c791f3-a10f-42e5-814f-8dba5f0e0cff
+
+Real-Time Demo:
+
+
+
+https://github.com/ethanzrd/whisper-playground/assets/79014814/0389ad47-ec62-4d6a-a6ff-aa23a8d3251f
```

Comment:
Haven't watched this one, but you can use the same audio you use for the sequential demo

Reply:
I think that the sequential demo should be enough - shouldn't it?